downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -369,6 +507,8 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 9 (2201.9.0) of
     }
     ```
 
+- To avoid clashes with Java identifiers, the character `$` which is used for encoding and decoding identifiers has been replaced by the character `&`.
```

Comment:
```suggestion
### Runtime changes

- To avoid clashes with Java identifiers, the character used for encoding and decoding identifiers has been changed from `$` to `&`.
```

We'll be doing the same for others in https://github.com/ballerina-platform/ballerina-dev-website/pull/8876/files.